File: SampleSetup/Output/TypeScript/SampleProject/Protocol/Body/ApiUserRegisterArg.ts

export class ApiUserRegisterArg {
    UserName: string = "";
    UserNameKind: number = 0;
    Password: string = "";

    FirstName: string | null = null;

    LastName: string = "";
    IsGuest: boolean = false;
    MobileMSISDN: string = "";
}

File: SampleSetup/Output/Kotlin/SampleProject/Protocol/Body/ApiUserRegisterArg.kt
package SampleProject.Protocol.Body

open class ApiUserRegisterArg {
    var UserName: String = ""

    var UserNameKind: Int = 0

    var Password: String = ""

    var FirstName: String? = null

    var LastName: String = ""

    var IsGuest: Boolean = false

    @Deprecated(message = "Please use MobileMSISDN")
    var Mobile: String = ""

    var MobileMSISDN: String = ""
}
